test/e2e/main_test.go

```diff
@@ -204,20 +224,26 @@ func TestExternalServiceWithAuth(t *testing.T) {
 		panic(err.Error())
 	}
 
-	okd.CreateRoute(Namespace, "cache-infinispan-0", "http")
+	okd.CreateRoute(Namespace, "cache-infinispan-0", 8080, "http")
 	defer okd.DeleteRoute(Namespace, "cache-infinispan-0-http")
 
+	okd.CreateRoute(Namespace, "cache-infinispan-0", 9990, "mgmt")
+	defer okd.DeleteRoute(Namespace, "cache-infinispan-0-mgmt")
+
 	client := &http.Client{}
 	hostAddr := okd.WaitForRoute(client, Namespace, "cache-infinispan-0-http", RouteTimeout, "connectorusr", "connectorpass")
 
+	hostAddrMgmt := okd.WaitForRoute(client, Namespace, "cache-infinispan-0-mgmt", RouteTimeout, "connectorusr", "connectorpass")
```

Comment:
Is this right? Waiting for a management service route to be available shouldn't require connector user/password. If anything it should need management user/pass, but I feel this method is not really using these credentials? Or under which circumstances does it use it?

Reply:
`checkExternalAddress` does a check wait basic authentication, I can't rememer why... need to check. I would go for simplifing this in a separated issue.

Reply:
Make sure you track that.

Reply:
https://github.com/infinispan/infinispan-operator/issues/95